docs/usage.md

```diff
@@ -46,7 +46,7 @@ The simulation mode has some sample data that helps you get a better understandi
 
     - CVE-2018-0175
 
-5. Navigate to **OT Vulnerability Management** > **ICS Advisory**. Following records are created:
+5. Navigate to **OT Vulnerability Management** > **ICS Advisory**. Following record is created:

```

Comment:
Asset name is : Stratix Services Router-4121 in line number 39. 

Unable to comment on the line in github.